web/react/components/admin_console/user_item.jsx

```diff
@@ -113,6 +113,8 @@ export default class UserItem extends React.Component {
         if (user.roles.length > 0) {
             if (user.roles.indexOf('system_admin') > -1) {
                 currentRoles = 'System Admin';
+            } else if (user.roles === 'admin') {
```

Comment:
Should be switched to use functions like Utils.isAdmin(roles), Utils.isSystemAdmin(roles), or Utils.isInRole(roles, inRole)